package.json

```diff
@@ -1,13 +1,15 @@
 {
   "name": "aff",
-  "version": "0.2.1",
+  "version": "0.2.0",
   "description": "Asynchronous effect system for TypeScript",
   "main": "out/index.js",
   "scripts": {
     "build": "tsc",
+    "install": "typings install",
+    "prepublish": "npm run build",
     "pretest": "npm run build",
-    "test": "mocha out/**/*-spec.js",
-    "prepublish": "npm run build"
+    "test": "./node_modules/.bin/mocha out/**/*-spec.js --recursive",
```

Comment:
mocha works, but it's an older version. With that command I make sure it's the version set in package.json